gallery_dl/extractor/4archive.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+"""Extractors for https://4archive.org/"""
+
+from .common import Extractor, Message
+from .. import text
+
+
+class _4archiveThreadExtractor(Extractor):
+    """Extractor for 4archive threads"""
+    category = "4archive"
+    subcategory = "thread"
+    directory_fmt = ("{category}", "{board}", "{thread} {title}")
+    filename_fmt = "{no} {filename}.{extension}"
+    archive_fmt = "{board}_{thread}_{no}"
+    pattern = r"(?:https?://)?4archive\.org/board/([^/?#]+)/thread/(\d+)"
+    root = "https://4archive.org"
+    example = "https://4archive.org/board/a/thread/12345/"
+
+    def __init__(self, match):
+        Extractor.__init__(self, match)
+        self.board, self.thread = match.groups()
+
+    def items(self):
+        url = "{}/board/{}/thread/{}".format(
+            self.root, self.board, self.thread)
+        page = self.request(url).text
+        data = self.metadata(page)
+        posts = self.posts(page)
+
+        if not data["title"]:
+            data["title"] = posts[0]["com"][:50]
+
+        for post in posts:
+            post.update(data)
+            post["time"] = text.parse_int(post["date"].timestamp())
```

Comment:
```suggestion
            post["time"] = int(util.datetime_to_timestamp(post["date"]))
```

[`datetime.timestamp()`](https://docs.python.org/3/library/datetime.html#datetime.datetime.timestamp) assumes "unaware" datetimes to be in local time, while `(post["date"]` is in UTC.